### PR TITLE
Update `pytest` arguments

### PR DESCRIPTION
**Pull request by mattwthompson**
Currently,

* `pytest-xdist` is installed but not leveraged
* test output is verbose for all tests, making it excessively long and difficult to parse

https://docs.pytest.org/en/8.0.x/how-to/output.html

### CODE REVIEW

**Comment by mattwthompson** [comment]
#263 would be nice to get in before this

**Comment by j-wags** [comment]
Unblocked by #263, but currently CI is failing for reasons that are likely on MolSSI's side. Given that xdist might cause its own concurrency issues, let's wait until we get green CI on main to test+merge this.

**Review by j-wags — APPROVED** [review]
LGTM!

**Pull request merged**
